src/chainstate/burn/db/sortdb.rs

```diff
@@ -2362,6 +2384,17 @@ impl SortitionDB {
         query_row(conn, qry, &args)
     }
 
+    /// Get a block commit's parent commit's block's sortition ID
```

Comment:
Perhaps more clear as 

```suggestion
    /// Get the Sortition ID for the burnchain block containing `block_commit_txid`'s parent
```

Reply:
Fixed